extensions/amp-date-countdown/validator-amp-date-countdown.protoascii

```diff
@@ -66,14 +66,19 @@ tags: {  # <amp-date-countdown>
   }
   attrs: {
     name: "timestamp-ms"
-    mandatory_oneof: "['end-date', 'timestamp-ms', 'timestamp-seconds']"
+    mandatory_oneof: "['end-date', 'timestamp-ms', 'timestamp-seconds','timeleft-ms']"
     value_regex: "\\d{13}"
   }
   attrs: {
     name: "timestamp-seconds"
-    mandatory_oneof: "['end-date', 'timestamp-ms', 'timestamp-seconds']"
+    mandatory_oneof: "['end-date', 'timestamp-ms', 'timestamp-seconds','timeleft-ms']"
     value_regex: "\\d{10}"
   }
+  attrs: {
```

Comment:
nit: we try to keep attributes in order alphabetically by name, so this should be higher up.